style.css

```diff
@@ -0,0 +1,161 @@
+body {
+  margin: 0;
+  font-family: sans-serif;
+  font-style: italic;
+}
+
+ul {
+  padding-left: 0;
+  list-style: none;
+}
+
+img {
+  max-width: 100%;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.section {
+  margin: 0;
+  overflow: auto;
+}
+
+.section--bg-pic {
+  margin: 0;
+  width: 100%;
+}
+
+.char-stats {
+  position: fixed;
+  bottom: 5rem;
+  left: 7rem;
+  color: white;
+  min-width: max-content;
+}
+
+.char-stats__img {
+  float: left;
+  width: 7rem;
+}
+
+.char-stats__health {
+  height: 10rem;
+  width: 12rem;
+  float: left;
+  position: relative;
+  top:1rem;
+  left: 0.5rem;
+}
+
+.char-stats__hb {
+  min-width: 4rem;
+  display: inline;
```

Comment:
Using `display: inline` along with width is making the border right hide. If you remove these values the right border will show up after your last health bar based on the div's content.